src/FsSpreadsheet/FsColumn.fs

```diff
@@ -0,0 +1,136 @@
+    new(index, (cells : FsCellsCollection)) = 
+        let getIndexBy (f : (FsCell -> int) -> seq<FsCell> -> FsCell) = 
+            match cells.GetCellsInColumn index |> Seq.length with
+            | 0 -> 1
+            | _ ->
+                (
+                    cells.GetCellsInColumn index 
+                    |> f (fun c -> c.Address.RowNumber)
+                ).Address.RowNumber
+        let minRowIndex = getIndexBy Seq.minBy
+        let maxRowIndex = getIndexBy Seq.maxBy
+        FsColumn (FsRangeAddress(FsAddress(minRowIndex, index),FsAddress(maxRowIndex, index)), cells)
+
+    interface IEnumerable<FsCell> with
+        member this.GetEnumerator() : System.Collections.Generic.IEnumerator<FsCell> = this.Cells.GetEnumerator()
+
+    interface IEnumerable with
+        member this.GetEnumerator() = (this :> IEnumerable<FsCell>).GetEnumerator() :> IEnumerator
+
+    // ----------
+    // PROPERTIES
+    // ----------
+
+    /// The associated FsCells.
+    member self.Cells = 
+        base.Cells(cells)
+
+    /// <summary>
+    /// The index of the FsColumn.
+    /// </summary>
+    member self.Index 
+        with get() = self.RangeAddress.FirstAddress.ColumnNumber
+        and set(i) = 
+            self.RangeAddress.FirstAddress.ColumnNumber <- i
+            self.RangeAddress.LastAddress.ColumnNumber <- i
+
+
+    // -------
+    // METHODS
+    // -------
+
+    /// <summary>
+    /// Creates a deep copy of this FsRow.
+    /// </summary>
+    member self.Copy() =
+        let ra = self.RangeAddress.Copy()
+        let cells = self.Cells |> Seq.map (fun c -> c.Copy())
+        let fcc = FsCellsCollection()
+        fcc.Add cells
+        FsColumn(ra, fcc)
+
+    /// <summary>
+    /// Returns a deep copy of a given FsColumn.
+    /// </summary>
+    static member copy (column : FsColumn) =
+        column.Copy()
+
+    /// <summary>
+    /// Returns the index of the given FsColumn.
+    /// </summary>
+    static member getIndex (column : FsColumn) = 
+        column.Index
+
+    /// <summary>
+    /// Returns the FsCell at rowIndex.
+    /// </summary>
+    member self.Cell(rowIndex) = 
+        base.Cell(FsAddress(rowIndex,1),cells)
+       
+    /// <summary>
+    /// Returns the FsCell at rowIndex.
+    /// </summary>
+    member this.Item (rowIndex) =
+        this.Cell(rowIndex)
+        //match _cells |> List.tryFind (fun cell -> cell.WorksheetColumn = columnIndex) with
+        //| Some cell ->
+        //    cell
+        //| None -> 
+        //    let cell = FsCell()
+        //    cell.WorksheetColumn <- columnIndex
+        //    cell.WorksheetRow <- _index
+        //    _cells <- List.append _cells [cell]
+        //    cell
+
+    /// <summary>
+    /// Returns the FsCell at the given rowIndex from an FsColumn.
+    /// </summary>
+    static member getCellAt rowIndex (column : FsColumn) =
```

Comment:
Though I found this name nice and descriptive it does not match F#.Core's style of getting items in IEnumerables.
See `List.item`, `Seq.item` and `Array.item` for comparison.

Therefore, we should name this `item`, too.